ovos_padatious/opm.py

```
@@ -13,26 +13,97 @@
 from ovos_bus_client.session import SessionManager, Session
-from ovos_config.config import Configuration
-from ovos_config.meta import get_xdg_base
+from ovos_padatious import IntentContainer as PadatiousIntentContainer
+from ovos_padatious.match_data import MatchData as PadatiousIntent
 from ovos_plugin_manager.templates.pipeline import ConfidenceMatcherPipeline, IntentHandlerMatch, IntentMatch
 from ovos_utils import flatten_list
 from ovos_utils.fakebus import FakeBus
 from ovos_utils.lang import standardize_lang_tag
 from ovos_utils.log import LOG, deprecated, log_deprecation
 from ovos_utils.xdg_utils import xdg_data_home
+from functools import lru_cache
```

Ruff (0.8.2)

Redefinition of unused `lru_cache` from line 18

Remove definition: `lru_cache`

(F811)